app/views/pages/data_requirements.html.erb

```diff
@@ -0,0 +1,87 @@
+<%= render PageTitle::View.new(title: "pages.data_requirements") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: 'Back',
+    href: "/home"
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <span class="govuk-heading-l govuk-!-margin-bottom-9">Check what data you need</span>
+
+    <p class="govuk-body">You’ll need the following information about a trainee to complete their record.</p>
+
+    <div class="govuk-heading-m">Personal Details</div>
+
+    <ul class="govuk-list">
+      <li>First names</li>
+      <li>Middle names</li>
+      <li>Last or family name</li>
+      <li>Date of birth</li>
+      <li>Gender</li>
+      <li>Nationality</li>
+    </ul>
+
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+    <div class="govuk-heading-m">Contact Details</div>
+
+    <ul class="govuk-list">
+      <li> Address </li>
+      <li> Email address</li>
+    </ul>
+
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+    <div class="govuk-heading-m">Diversity information (optional)</div>
+    
+    <div class="govuk-inset-text"> If the trainee chose not to share this information during the application process, do not ask for it for the purpose of registering them.</div>
+
+    <ul class="govuk-list">
+      <li>Ethnicity</li>
+      <li>Disability</li>
+    </ul>
+
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+    <div class="govuk-heading-m">Degrees</div>
+
+    <div class="govuk-heading-s">If it's a UK degree</div>
+
+    <ul class="govuk-list">
+      <li>Type</li>
```

Comment:
Should read: Type (for example, BA, BSc or other)

Reply:
thanks, fixed